docs/en/operations/settings/settings.md

```diff
@@ -1455,6 +1455,17 @@ See also:
 -   [insert_quorum_timeout](#settings-insert_quorum_timeout)
 -   [insert_quorum_parallel](#settings-insert_quorum_parallel)
 
+## insert_keeper_max_retries
```

Comment:
Well, this description is actually not correct. Let me add keeper retries settings in other PR, there are several of them. Feel free to remove `insert_keeper_max_retries` from this PR and merge rest minor changes

Reply:
See https://github.com/ClickHouse/ClickHouse/pull/45788